devel/sync-mainnet/sync-mainnet.yaml

```diff
@@ -1,14 +1,36 @@
 start:
   args:
-  - relayer
-  - merger
-  - reader-node
+    - merger
+    - reader-node
+    - relayer
+    - firehose
+    - combined-index-builder
   flags:
     common-chain-id: 1
     common-network-id: 1
     common-live-blocks-addr: :23011
```

Comment:
I'm actually now thinking that this should be removed so that the firehose grpc server is available prior to the whole blockchain being synced. Thoughts?

Reply:
It should be able now to deal with non real-time live sources, if it does not work, we can take a look.